webui/react/src/pages/ExperimentDetails/ExperimentDetailsHeader.tsx

```diff
@@ -517,6 +518,11 @@ const ExperimentDetailsHeader: React.FC<Props> = ({
             <div className={css.name} role="experimentName">
               {experiment.name}
             </div>
+            {experiment.unmanaged && (
+              <Badge tooltip="Workload not managed by Determined" type="Header">
```

Comment:
Suggestion: different color is more visible in dark mode

<img width="504" alt="image" src="https://github.com/determined-ai/determined/assets/109113616/a35cc03c-a935-429c-b844-3639370a975c">

Reply:
<img width="940" alt="Screenshot 2023-09-11 at 4 29 36 PM" src="https://github.com/determined-ai/determined/assets/40620519/d83d4287-7d61-475c-8e88-781eeda11e92">
Border added as suggested by Michiel